pulser-core/pulser/json/abstract_repr/serializer.py

```diff
@@ -153,14 +153,7 @@ def convert_targets(
         og_dim = target_array.ndim
         if og_dim == 0:
             target_array = target_array[np.newaxis]
-        try:
-            indices = seq.register.find_indices(target_array.tolist())
-        # RuntimeError raised when calling seq.register for a MappableRegister
-        except RuntimeError:
-            raise NotImplementedError(
-                "Serialization of sequences with local operations and"
-                " a mappable register is currently not supported."
-            )
+        indices = seq._register.find_indices(target_array.tolist())
```

Comment:
Sweet! Once `Sequence.get_register()` is defined, it would be nice to use it here instead of accessing `_register`

Reply:
`Sequence.get_register()` is now added ;)